soroban/README.md

```diff
@@ -7,6 +7,7 @@ This repository contains [Soroban] contracts used in the `sep-smart-wallet` appl
 
 ## Contracts
 
+- `webauth-verify`: A Soroban contract used by a SEP-10c server to construct a challenge for a WebAuthn device to verify its ownership of a Soroban account contract. This is not used by this project, but is included for reference.
```

Comment:
💡  It may be worth it to include the permalink to the AP branch where it's being used. WDYT?

Reply:
Good idea. I've updated the README.